CHANGELOG.md

```diff
@@ -3,6 +3,17 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and
 the [“Keep a Changelog” format](http://keepachangelog.com/).
 
+
+## 3.1.0 - 2016-11-15
```

Comment:
Use “Upcoming” as the title. See the [DEVELOPING.md](https://github.com/contentful/ui-extensions-sdk/blob/master/DEVELOPING.md).